big_tests/tests/gdpr_SUITE.erl

```diff
@@ -175,6 +196,9 @@ pick_backend_for_mam() ->
 mam_required_modules(Backend) ->
     [{mod_mam_meta, [{backend, Backend}, {pm, []}]}].
 
+vcard_required_modules() ->
+    [{mod_vcard, [{backend, mnesia}]}].
```

Comment:
What about RDBMS? Riak? I'd prefer to see `dynamic_modules:save_modules/2` and `dynamic_modules:restore_modules/2` used in "killer" groups init and end cleanup.

Reply:
Ah, yes, forgot to use different backends in tests

Reply:
Are you saing to move `save_modules` from `init_per_suite` to `init_per_group` ( and vice versa for restore) ?